crates/telio-model/src/features.rs

```diff
@@ -536,6 +540,7 @@ mod tests {
             "derp": {
                 "tcp_keepalive": 13,
                 "derp_keepalive": 14,
+                "poll_keepalive": true,
```

Comment:
add a test that verifies what will be deserialized if the field is missing

Reply:
It already exists `test_empty_derp`